mailer.js

```diff
@@ -68,10 +68,13 @@ module.exports = function (log) {
     if (message.resume) { query.resume = message.resume }
 
     var link = this.verificationUrl + '?' + qs.stringify(query)
+    query.one_click = "true"
```

Comment:
can we safely make this a true `true` instead of a string?
it gets `qs.stringify(query)`'d anyway